### DISCUSSION TITLE
Ollama and ChatGPT not responding through Remote PC but works fine on local

### DISCUSSION THREAD

**Discussion by TrippyToes** [open-webui/open-webui#3925] [Q&A]
Hi All,

I've started tinkering with Ollama and OpenWebUI. Please note I'm not a very technical person but I have some basic understanding of docker and networking.

I'm trying to use OpenWebUI with Ollama in my PC via docker containers.

It works great if I access it within the same PC, however, if I try to access it remotely, I run into some problems.

![image](https://github.com/user-attachments/assets/cb06d66a-04a1-4ebd-ad79-c7f7006af3f3)

The Issue, Based on the above diagram:

**From the Host PC (PC B)** - no issues here, I can login start a new chat and receive responses from Ollama, use all features, etc.

**From remote devices (PC A, Tablet, Mobile)** - I can always reach OpenWebUI and use most features within the UI such as create users, do web searches, I can verify the connection to Ollama via the admin page, download new models to Ollama via the admin page, see installed models from the chat model selection dropdown, etc. **However**, I can't start a chat with any model. whenever I send the message, the model never responds. if I check the Ollama logs, there is no request received from the remote devices

I don't know whether the issue is with Ollama or OpenWebUI. some assistance here would be appreciated since I've been trying to troubleshoot this for a couple of days

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/80e6d58f-3500-401c-8a3e-a5990cc9ac3a">

 

![image](https://github.com/user-attachments/assets/df25a92d-f33b-4757-9ca2-a3e4474f3444)

<img width="624" alt="image" src="https://github.com/user-attachments/assets/25a1d7b7-f75d-4742-a1df-d7dc8620e35f">

UPDATE 1 ---

I subscribed to the chatGPT API plan to test and there is a similar behaviour, it works well from the host machine but if I try from a remote PC I get the following:

![image](https://github.com/user-attachments/assets/811a38ec-2c69-4b40-be8c-ad5cba508b19)

----
Environment
Open WebUI Version: 0.3.8

Ollama (if applicable): 0.2.5

Operating System: Windows 11

Docker: 4.32.0

Browser (if applicable): Chrome, Brave, Edge

**Comment by dugiahuy** [comment]
I also faced similar issue

**Reply by Jerey004** [reply]
Same issue as well

**Reply by thtmnisamnstr** [reply]
I'm running into this issue as well.

**Comment by revoconner** [comment]
Yeah same here, ollama and openwebui is running natively on windows using python 3.11

**Comment by thtmnisamnstr** [comment]
I found a solution for my issue (Windows 11). In Settings, go to Network & Internet. At the top, it should show tiles for your network/internet connection, Properties, and Data Usage. If Properties is set to Public, you need to change it to Private. You can do that by clicking on the Properties tile and changing your Network Profile Type to Private.

**Comment by thtmnisamnstr** [comment]
It looks like [v0.5.18](https://github.com/open-webui/open-webui/releases/tag/v0.5.18) may have fixed this issue.
